src/rewards/voters.rs

```diff
@@ -33,14 +37,65 @@ pub fn calculate_stake<'address>(
 pub fn calculate_reward_share<'address>(
     total_stake: u64,
     stake_per_voter: &HashMap<&'address Address, u64>,
+    threshold_addresses: &AddressesVoteCount,
+    threshold: u64,
 ) -> HashMap<&'address Address, U64F64> {
```

Comment:
unrelated: maybe we should use an alias for `U64F64` like we did in ca rewards

Reply:
Yep, I'll do.